Formula/aqbanking.rb

```diff
@@ -31,7 +31,6 @@ class Aqbanking < Formula
   def install
     ENV.deparallelize
     inreplace "aqbanking-config.in.in", "@PKG_CONFIG@", "pkg-config"
-    system "autoreconf", "-fiv" if build.head?
```

Comment:
There is no `head` spec, so this can be removed.

Reply:
👍